docs/concepts/governance/02-process.md

```diff
@@ -3,92 +3,109 @@ id: process
 title: Process
 ---
 
-This document is a living document which represents the current process guidelines for developing and advancing Uniswap Governance Proposals.
+This is a living document which represents the current process guidelines for developing and advancing Uniswap Governance Proposals. It was last updated September 2024.
 
-## Process
+## Tools
 
-Several governance venues are available to Uniswap governance, each serving its own particular purpose.
+There are several venues where Uniswap Governance takes place. Each serves its own particular purpose.
 
-1. [_gov.uniswap.org_](https://gov.uniswap.org/)
+1.  [_Governance Forum_](https://gov.uniswap.org/)
 
-gov.uniswap.org is a Discourse-hosted forum for governance-related discussion. Community members must register for an account before sharing or liking posts. New members are required to enter 4 topics and read 15 posts over the course of 10 minutes before they are permitted to post themselves.
+A Discourse-hosted forum for governance-related discussion. Community members must register for an account before sharing or liking posts. New members must read 4 topics and a combined 15 posts over the course of at least 10 minutes before they may  post themselves.
 
 2. [_Snapshot_](https://snapshot.org/#/uniswapgovernance.eth)
 
-Snapshot is a simple voting interface that allows users to signal sentiment off-chain. Votes on snapshot are weighted by the number of UNI delegated to the address used to vote.
+A simple voting interface that allows users to signal sentiment off-chain. Votes on Snapshot are weighted by the number of UNI delegated to the address used to vote.
+
+1. [_Uniswap Agora_](https://vote.uniswapfoundation.org)
 
-3. [_Governance Portal_](https://app.uniswap.org/#/vote)
+The [Uniswap Foundation](https://www.uniswapfoundation.org) supports this voting and delegation interface. (Tally)[https://www.tally.xyz/gov/uniswap] is another app that supports proposal creation, delegation, and voting.
```

Comment:
I didn't see a link on the main page pointing to the vote.uniswapfoundation.org. Is this intentional, should the link above point to the vote page

Reply:
Nit:
in a few places, you use threshold instead of Quorum.  However only quorum is defined, wondering if it should be all quorum, or add threshold to the area defining quorum

Reply:
good shout on quorum / threshold - will clarify.

re main page, do you mean on uniswap.org? Ideally would love to get every link to point to `vote.uniswapfoundation.org` as `app.uniswap.org/vote` is deprecated.

Reply:
Sorry didn't think about having both links there. 
the link you have goes to the foundation home page with the www. I didn't see a link to the subdomain vote.uniswapfoundation.org

Reply:
Looks like there was a PR merged between this, just FYI to update your branch to reflect

Reply:
so re the links yea, we're linking to both the voting interface (vote.uniswapfoundation.org) and the foundation website (uniswapfoundation.org

i'll get www.uniswapfoundation.org to link out to the voting interface

Reply:
@Rachel-Eichenberger thanks for your nits! think content is done. re updates to docs doesn't look like i have any merge conflicts so should be okay